lib/lunchbot_web/live/order_live.ex

```diff
@@ -0,0 +1,131 @@
+defmodule LunchbotWeb.OrderLive do
+  use LunchbotWeb, :live_view
+
+  alias LunchbotWeb.Router.Helpers, as: Routes
+
+  alias Lunchbot.Repo
+
+  import Ecto.Query
+
+  alias Lunchbot.LunchbotData
```

Comment:
Should all the alias fns be grouped together?  Then you could import Ecto.Query below them.